### ISSUE TITLE
Shift the 'Damage Test' into Level 50

### ISSUE THREAD

**Issue by nvs** [nvs/gem#90]
Yup, a test of your maze's damage, nothing more. It should honestly not be possible to kill it, and things should be balanced around that fact. On top of that, it should probably be more than one unit, to be fair to mazes that lean away from pure focus fire.

Now, if someone kills the test level, then they should be rewarded. Perhaps a section in the Information area that congratulates these players, when provided sufficient proof. And then after that, balance whatever strategy they used because it is obviously too strong! Of course, this reward would not be implemented until further balancing is done and things are a bit more stable.

Sarcasm aside, this actually is a good idea. Yes, it means the race would end after finishing the round just before the `Damage Test`. Ideally, there would be multiple viable strategies. This obviously wouldn't be put in right away.

**Comment by nvs** [comment]
Probably the best solution would be to have multiple 'Final Boss' units, and if a player beats that round then they are given a menu of options. One of which can be to run the 'Damage Test'. Basically, make it clear to everyone that the race is officially over, and everything from a certain point on is just extra.

**Comment by nvs** [comment]
For now, the Damage Test will continue to start immediately after finishing 'Round 50'. However, it will be changed.
- Instead of a single unit, it will now consist of three units.
- The HP will be lowered by half, so each will have one million hit points, instead of two million.

The above may need to be tweaked. My goal is to have a 'Damage Test' round that seems possible but is actually impossible no matter what strategy is used.

**Comment by nvs** [comment]
Blocked by #137.

**Comment by nvs** [comment]
Rather than introducing multiple Damage Test units the hit points will simply be increased. This should be more acceptable to existing players, while making it a harder event.

**Comment by nvs** [comment]
This is effectively blocked by the stun lock effectiveness via Ancients increasing, as mentioned in #253. Until then no changes to the Damage Test will be made.

**Comment by nvs** [comment]
Probably shift the Damage Test into Level 50 proper. Basically, making it a boss. Have a few issues related to this now, I'm sure.

**Comment by nvs** [comment]
In short, 1.6 will not have a Damage Test. The eventual goal is to have the Damage Test(s) be separate from the main game anyway; a bonus that one can participate in due to beating the boss. We're not there yet. But, we can have the real (harder) boss. The existing Damage Test is a joke. And since stunlock is not going to disappear, the design of the 'final' boss has to be changed. So here we are.

Probably a cool million HP on the boss will keep most heads level. Too low, and people will cry it is too easy. Too high, and people will cry because they cannot finish the game as often.

Still, the alternative of turning the 'boss' into multiple boss units is a possibility. This would leave Level 50 as is, and simply make 51 the official boss instead.

**Comment by nvs** [comment]
Rather than shove the Damage Test into Level 50, we'll just split it into multiple units. Easy peasy. *I can still hear the cries of players in the distance...*
